visualization/app/codeCharta/ui/codeMap/codeMap.arrow.service.ts

```diff
@@ -77,6 +77,7 @@ export class CodeMapArrowService implements BuildingHoveredSubscriber, BuildingU
 				this.addArrow(targetNode, originNode)
 			}
 		}
+		this.threeSceneService.highlightBuildings()
```

Comment:
I guess this line adding is a mistake, isn't it?

Reply:
I had to add that line there because otherwise the higlightingeffect would be deployed each time an edge is found for the currently hovered building. But when I add that line after the for loop, the highlighting would only deploy once, after all the buildings with an edge to the hovered building is inside  the list.

Reply:
Okey after having a look at the code, I got it. So: 
- for every edge, the originNode or targetNode is added to the list of highlightedBuildings inside the threeSceneScervice
- and once after every node is added (after the for-loop) the added buildings get highlighted